CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 0.1.12
+
+### Added
+
+- `actions/create_host.yaml` - register a new host object to zabbix server.
```

Comment:
I think we're stuck in `v0.1.x` version.
Patch release makes sense for the bugfixes, while minor version change is for any new non-breaking additions or improvements.

To follow the https://semver.org/#summary, do you think we can assign `0.2.0` tag for this change and follow this scheme?

Reply:
> To follow the https://semver.org/#summary, do you think we can assign 0.2.0 tag for this change and follow this scheme?

I got it. I'll develop packs by conforming to this standard from now. Thank you for telling me.